src/architecture/configuration.md

```diff
@@ -78,6 +78,8 @@ two parts:
   Timeout interval (ms) for sending a `Status` message
 - **txs_block_limit**  
   Maximum number of transactions per block
+- **max_message_len**
+  Maximum message length (in bytes)
```

Comment:
Would you mind adding some details? If I understand correctly, something like this:

> The parameter decides the maximum allowed size of both consensus messages and transactions. The default value of the parameter is 1 MiB, i.e., `1024 * 1024` bytes.

Please consult developers for the sane range of the values. I *think* it's somewhere around 1 MiB - 16 MiB, but I'm not completely sure.